src/app/menu-bar/menu-bar.component.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>TaskManager</title>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <nav class="navbar navbar-inverse">
```

Comment:
Please remove all redundant code here and leave the only navigation

Reply:
all  redundant  code is removed.